packages/react-examples/src/react/DetailsList/DetailsList.ColumnResize.Example.tsx

```diff
@@ -0,0 +1,120 @@
+import * as React from 'react';
```

Comment:
Thanks for adding an example! This will also need to be added in a couple other places to show up on the site (because our site is not the smartest...):
- `packages/react-examples/src/react/DetailsList/DetailsList.doc.tsx`
- Files similar to `apps/public-docsite/src/pages/Controls/DetailsListPage/DetailsListAdvancedPage.tsx` and `DetailsListAdvancedPage.doc.ts` (in the same directory)
- A file similar to `apps/public-docsite-resources/src/components/pages/DetailsList/DetailsListAdvancedPage.tsx`

Reply:
This example is less of a showcase and more for validation. I'm not sure it's ready yet to go on the doc site.